homeassistant/components/light/zha.py

```diff
@@ -73,7 +73,34 @@
                         UNSUPPORTED_ATTRIBUTE):
                     discovery_info['color_capabilities'] |= \
                         CAPABILITIES_COLOR_TEMP
-        entities.append(Light(**discovery_info))
+
+        light_entity = Light(**discovery_info)
+
+        if discovery_info['new_join']:
+            from zigpy.zcl.clusters.general import OnOff, LevelControl
+            in_clusters = discovery_info['in_clusters']
+            if OnOff.cluster_id in in_clusters:
```

Comment:
But really for this section of code we should be using https://github.com/home-assistant/home-assistant/pull/19177 :)

Reply:
As soon as it’s merged....